dotnet-logging/LogLevelsWithSerilog/LogLevelsWithSerilog/Program.cs

```diff
@@ -0,0 +1,49 @@
+using Serilog;
+using Serilog.Sinks.MSSqlServer;
+
+IConfigurationRoot configuration = new ConfigurationBuilder()
+           .SetBasePath(Directory.GetCurrentDirectory())
+           .AddJsonFile("appsettings.json")
+           .AddJsonFile("appsettings.Development.json")
+           .Build();
+
+Log.Information("Web Host started");
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddSerilog(options =>
+        {
+            //we can configure serilog from configuration
+            options.ReadFrom.Configuration(configuration);
+
+            //or we can configure serilog via fluent api
+            options.MinimumLevel.Information()
+                   .WriteTo.Console(Serilog.Events.LogEventLevel.Warning, outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}] {Message:lj}{NewLine}{Exception}")
+                   .WriteTo.File("logs/myapplogs.txt",
+                        rollingInterval: RollingInterval.Day,
+                        rollOnFileSizeLimit: true,
+                        restrictedToMinimumLevel: Serilog.Events.LogEventLevel.Information)
+                   .WriteTo.MSSqlServer(
+                        connectionString: "Server=localhost;Database=LogDb;Integrated Security=SSPI;",
+                        sinkOptions: new MSSqlServerSinkOptions { TableName = "Logs" },
```

Comment:
This will be in the review document as well, but let's remove the SQL server logging from the example as we are not really talking about it in the article and only showing it in the configuration part.

Reply:
In my article, under "Configure Log Targets ..." sections i provided sample source codes about MSSqlServer sinks. Also under "Where to Log?" section i said some come sinks => "Relational databases, like MSSqlServer". Therefore, i think it will be a good choice to leave configuration details about this sink both in code and in the article. But If your opinion hasn't changed just tell me and then i will remove form code.

Reply:
My opinion hasn't changed and I mention it with a little more detail in the review document, but basically we are not talking about it in the section where we talk about differentiating log sinks based on log level. We are only showing it here in the configuration section, but not really talking about it in the article outside of that context. True we mention that as a potential sink in that section, but we also mention MongoDB under Non-relational DBs. Since we are not talking about the SqlServer part in the differentiation section, we don't need to include it here either. It helps to shorten the code snippet in the article and helps us focus on what the main point of the article is - logging messages to multiple sinks but at different levels. This is shown clearly with just the two sinks - Console and File. Our goal is to make the example as simple as possible while still demonstrating the topic completely.

Reply:
Ok, i will update the code.

Reply:
I  updated both the article and code and removed the MSSqlServer code samples.